### DISCUSSION TITLE
Add a New Project About Our Documentation

### DISCUSSION THREAD

**Discussion by ghost** [pulsar-edit/.github#30] [General]
> **Note**
> Not related to [this issue shown on profile README](https://github.com/pulsar-edit/pulsar-edit.github.io/issues/14).

I feel like we should have a new project hosted on our GitHub organization about our documentation. That way, people can get to what is in progress, to-do, and complete.

**If I get mostly "Yes" answers 4 days from now, I will create the project. If someone else creates it before then, I'll create a new project on my behalf.** (*Edit 1: I will pick a random time now.*)

**Comment by Daeraxa** [comment]
Would it make sense to link it to #14 (and related issues that go into more detail like #35)

**Reply by confused-Techie** [reply]
Personally I'd say to just reimplement whats available on #14 instead of just linking to it

**Reply by ghost** [reply]
> Personally I'd say to just reimplement whats available on #14 instead of just linking to it

You probably didn't see this:

> **Note**
> Not related to [this issue shown on profile README](https://github.com/pulsar-edit/pulsar-edit.github.io/issues/14).

**Reply by ghost** [reply]
Also it can be used for everything else, not simple reimplementation.

**Reply by confused-Techie** [reply]
Ahh @softcode589 my bad, I did not, somehow. But even without that, maybe more go with the previous suggestion as something can link to it. But then I'm assuming this project would contain larger scale actions, like getting templates removed from repos, changing repo descriptions and that sort of thing. Would be very helpful to get it all down in one place

**Reply by ghost** [reply]
> Ahh @softcode589 my bad, I did not, somehow. But even without that, maybe more go with the previous suggestion as something can link to it. But then I'm assuming this project would contain larger scale actions, like getting templates removed from repos, changing repo descriptions and that sort of thing. Would be very helpful to get it all down in one place

No... I'm proposing items for large-scale, medium-scale, and small-scale actions. (Except actions that aren't really needed to be in a project.)

**Reply by confused-Techie** [reply]
Fair enough. But yeah sounds good to me. Really its the only thing missing a Project board that we are working on then. So sounds fantastic

**Reply by Daeraxa** [reply]
I think there needs to be a clear overview of what is being proposed to put into the project and its layout. For example you could have one board for the entire project and views dedicated to each section (like a view that only has issues like #14) or a view that has the separated items like #35.

**Reply by ghost** [reply]
> I think there needs to be a clear overview of what is being proposed to put into the project and its layout. For example you could have one board for the entire project and views dedicated to each section (like a view that only has issues like #14) or a view that has the separated items like #35.

I don't really like views dedicated to each section. I like one, dedicated view of the roadmap about everything documentation at Pulsar. Just like what @confused-Techie said, getting it down to one place.

**Reply by ghost** [reply]
Now I feel like doing it now.

**Reply by kaosine** [reply]
@Daeraxa yea that was my plan to update that when possible. It just didn't get done so far since we're still so far back on finishing those to any level of true completion. Also lost where that was for a bit...

**Reply by Daeraxa** [reply]
I'm not voting no, I just think it would be useful to see a proposed outline to see if it is something we actually want to work with. There isn't much point to it if only one person uses it.

**Reply by kaosine** [reply]
I also meant to convert that topic to a project board. I just forgot.

**Reply by ghost** [reply]
Everyone I feel like everyone needs to help on the board (Private) https://github.com/orgs/pulsar-edit/projects/6/views/1

**Reply by Daeraxa** [reply]
Maybe ditch the chapters/groups and prefix the individual sections with the chapter/group

**Reply by kaosine** [reply]
That's what I planned on doing. I hated that chapter view and wanted to fix it
